libvast/test/meta_index.cpp

```diff
@@ -121,7 +121,7 @@ struct fixture {
     for (size_t i = 0; i < num_partitions; ++i) {
       auto name = i % 2 == 0 ? "foo"s : "foobar"s;
       auto& part = mock_partitions.emplace_back(std::move(name), ids[i], i);
-      auto ps = make_partition_synopsis(part.slice, meta_idx.factory_options());
+      auto ps = make_partition_synopsis(part.slice, {});
```

Comment:
You can also remove the second argument, this is a helper function just for this unit test.

Reply:
Thanks, totally missed that.